### PR TITLE
fix release process

### PR DESCRIPTION
**Pull request by anthonyroach**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -2,33 +2,27 @@ name: Release
 
 on:
   push:
-    branches:
```
Comment (anthonyroach) [inline .github/workflows/release.yml:5]: these push conditions are ORed, so the job runs on all pushes to master and all v* tags on any branch

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -2,33 +2,27 @@ name: Release
...
     tags:
       - "v*"
 
 jobs:
   build:
+    if: ${{ github.head_ref }} == "master"
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Exit if not on master branch
```
Comment (anthonyroach) [inline .github/workflows/release.yml:15]: this doubly doesn't work:

problem #1: `github.ref` is the tag ref, not the branch
problem #2: `exit 0`  doesn't exit the action, just the run block

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -2,33 +2,27 @@ name: Release
...
-        if: endsWith(github.ref, 'master') == false
-        run: exit 0
       - uses: actions/setup-node@v1
         with:
           node-version: 12
           registry-url: https://registry.npmjs.org
       - name: Build and npm publish
-        if: startsWith(github.ref, 'refs/tags/v')
```
Comment (anthonyroach) [inline .github/workflows/release.yml:23]: This is not needed now that the trigger is just v* tag creates

**Review by ynotdraw — APPROVED** [review]
Cool. We haven't touched this since MarkS and I set all of this up when we were first getting into GitHub actions.  We can probably apply this to chroma as well, since the config was copy/pasted over.

**Review by joedimarzio — APPROVED** [review]

**Comment by anthonyroach** [comment]
> Cool. We haven't touched this since MarkS and I set all of this up when we were first getting into GitHub actions. We can probably apply this to chroma as well, since the config was copy/pasted over.

already done

**Pull request merged**